### ISSUE TITLE
Empty sequence error

### ISSUE THREAD

**Issue by martindholmes** [projectEndings/diagnostics#10]
Just test-running against the MoEML /db/data folder, I got this:

~~~~~
 [java] Checking website_structure.xml (219/1585)
     [java] Type error at char 16 in xsl:variable/@select on line 438 column 49 of diagnostics_master.xsl:
     [java]   XTTE0570: An empty sequence is not allowed as the value of variable $thisAttTokens
     [java] An empty sequence is not allowed as the value of variable $thisAttTokens
     [java] Java Result: 2
~~~~~

Seems like it might be a whitespace-normalization issue or something like that. No time to debug it right now.

**Comment by joeytakeda** [comment]
I added the standalone version of this file to the test directory. I think it's this line causing it, which might mean there is a bug with the namespace check. Investigating now.

`<egXML
                xmlns="http://www.tei-c.org/ns/Examples" valid="true">
                <facsimile>
                  <surface>
                    <graphic url=""/>
                    <zone/>
                  </surface>
                </facsimile>
              </egXML>`

**Comment by joeytakeda** [comment]
I spoke too soon--I was using the old version of the code that wasn't checking for namespaces. The namespace selector seems to be working fine. Was your build running the code before you added namespaces?

I added an example of `<p next=""/>` to the corpus document, which caused the build to fail. Will be a quick fix.

**Comment by joeytakeda** [comment]
Fixed in commit 67c6bdd.

**Comment by martindholmes** [comment]
At first I thought "wait a minute -- an empty pointer is a broken pointer", but then I realized that we don't actually know whether an att val is supposed to be a pointer or not, and it's perfectly OK for many atts to have empty values.
